.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/administrate
+    docker:
+      - image: circleci/ruby:2.4.1-node-browsers
+        environment:
+          PGHOST: 127.0.0.1
+          PGUSER: administrate
+          RAILS_ENV: test
+      - image: postgres:9.5-alpine
+        environment:
+          POSTGRES_USER: administrate
+          POSTGRES_DB: administrate-prototype_test
+          POSTGRES_PASSWORD: ""
+    steps:
+      - checkout
+
+      # Restore Cached Dependencies
+      - type: cache-restore
+        name: Restore bundle cache
+        key: administrate-{{ checksum "Gemfile.lock" }}
+
+      # Bundle install dependencies
+      - run: bundle install --path vendor/bundle
+
+      # Install Appraisal gemfiles
+      - run: bundle exec appraisal install
+
+      # Cache Dependencies
+      - type: cache-save
+        name: Store bundle cache
+        key: houndci-{{ checksum "Gemfile.lock" }}
```

Comment:
Should the key for this cache be the same as the one on line 22?

Reply:
Yep, that's pretty much needed @nickcharlton should be `administrate-{{ checksum "Gemfile.lock" }}`

Reply:
Ah, yeah. I missed that.